### DISCUSSION TITLE
User/Installation guide?

### DISCUSSION THREAD

**Discussion by llGaetanll** [warpwm/lule#2] [Q&A]
Hey this project seems really cool, I'd love to use it but could not find any info on how to do so.

**Comment by bresilla** [comment]
Hey, i am glad you like it.

Well, for the moment, the only way to get it is to clone locally, then build it with: `cargo build --release`. Which means you need to have `cargo` and `rustc` installed in your system.

Hopefully soon (if i have the time, I'll setup a CI for automatic builds and you can simply grab the binary)...

**Reply by llGaetanll** [reply]
I got it to work, but I should mention that I had to download the git submodules before I could `cargo build --release` the application which might not be intuitive for everyone. In any case, if anybody else is looking to use this, make sure you
```
git submodule update --init --recursive
cargo build --release
```

**Reply by bresilla** [reply]
Yeahhh, thats true. I have forgotten about it. 
I use a library that i needed some addition functionality to implement on it, thus i have as submodule.

I will as soon as i can, make documentations about everything.

P.S. if you check the help (--help), this app includes quite a lot. but i have documented nothing still, and I still want to add a lot more functionality... Because for the moment is half-baked :/

But, please, if you find and bugs, or even better, if you wish to add more functionality, just open an issue, and i will gladly look at it and fix (if bug) or implement (if feature request).

Cheers.

**Reply by bresilla** [reply]
And thanks :)
